src/HtmlUtils.tsx

```diff
@@ -414,6 +418,21 @@ export function bodyToHtml(content: IContent, highlights: string[], opts: IOpts
         if (isHtmlMessage) {
             isDisplayedWithHtml = true;
             safeBody = sanitizeHtml(formattedBody, sanitizeParams);
+            const phtml = cheerio.load(safeBody,
```

Comment:
Let's move this inside the settings block, so we can skip this work if the feature is not enabled.